backend/store/etcd/entity_store.go

```diff
@@ -5,10 +5,10 @@ import (
 	"encoding/json"
 	"errors"
 	"fmt"
-	"path"
 
 	"github.com/coreos/etcd/clientv3"
 	"github.com/sensu/sensu-go/backend/store"
+	"github.com/sensu/sensu-go/types"
```

Comment:
It's OK not to change now, but as a reminder for future changes/refactors: we have type aliases to move away from the `types` package. We want to use the `corev2 "github.com/sensu/sensu-go/api/core/v2` package instead.